lib/compat/wordpress-6.1/template-parts-screen.php

```diff
@@ -103,6 +103,8 @@ static function( $classes ) {
 		}
 	);
 
+	$extra_template_slugs = apply_filters('extra_template_slugs', []);
```

Comment:
We would probably need more than slugs, like we do for the default template types. For example at least a description I think will be needed.

Reply:
Changed to use template types instead of slugs 👍